### ISSUE TITLE
Relax validation on the uniqueness of parliamentary_term_items

### ISSUE THREAD

**Issue by siwhitehouse** [mysociety/verification-pages#146]
https://github.com/mysociety/verification-pages/blob/master/app/models/page.rb#L10

enforces uniqueness on the legislative terms that can be used on Validation Pages. However, we have the possibility that the same legislative terms may be used for the head of government and ordinary members. After discussing this with Rich and Graeme we decided that this validation rule should be relaxed.

**Comment by tmtmtmtm** [comment]
> the same legislative terms may be used for the head of government and ordinary members

I don't really have much context for this, so may be misunderstanding, but the head of government, as executive-level position, doesn't usually have a *legislative* term. In Wikidata there are different properties for these: P2937 is for legislative terms, and P5054 is the equivalent for the executive position. See, for example https://www.wikidata.org/wiki/Q388442#P39

**Comment by siwhitehouse** [comment]
Thanks Tony - the context is the mayors of (at least some) mayors in Italy. I have removed the legislative term and electoral district qualifiers from https://www.wikidata.org/wiki/Q24701812, edited the verification page for the Mayor of Turin by removing the term and then I have been able to create the verification pages for the city council members.
Based on this I don't think we want/need to remove the validation.
